specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json

```diff
@@ -1146,12 +1146,12 @@
       "properties": {
         "name": {
           "type": "string",
-          "readOnly": true,
+          "readOnly": false,
           "description": "Name of the configuration."
         },
         "version": {
           "type": "string",
-          "readOnly": true,
+          "readOnly": false,
           "description": "Version of the configuration."
         }
       },
```

Comment:
As Michael mentioned, seems the sub property "[configurationMode](https://github.com/Azure/azure-rest-api-specs/blob/08faf3284d23b2e4d22568563281d14ff0956529/specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json#L845)" of the configurationSetting property is writeable, right? If yes, seems it hasn't been updated to writeable in this PR, right?

Reply:
That correct. AssignmentType is not part of this PR